### PR TITLE
Trace page performance with Sentry

### PR DESCRIPTION
**Pull request by eatyourgreens**
- install `@sentry/tracing` in both NextJS apps.
- enable performance tracing in Express and in the browser.
- turn off Sentry in local development mode (`NODE_ENV = 'development'`) so that Sentry isn't spammed with empty development releases.

## Package
app-project
app-content-pages

## How to Review
There are a few options. Pick the one you're most comfortable with:
- [Build and start both NextJS apps locally in Node](https://github.com/zooniverse/front-end-monorepo/#with-node-and-yarn) to verify that they still build and run.
- [Build and start a Docker image](https://github.com/zooniverse/front-end-monorepo/#docker), then check both NextJS apps on localhost. This will verify that a production install (no dev dependencies) starts and runs in a container.
- [Build and deploy a Docker image of this branch](https://github.com/zooniverse/front-end-monorepo/actions/workflows/deploy_branch.yml), then check the project app on https://fe-project-branch.preview.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess. There's no branch preview on kubernetes for the content pages app.

Production builds will log performance tracing information to Sentry, while you are browsing a project (except in Brave, which blocks all tracking.) Local development mode (`yarn dev`) won't log to Sentry at all.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [x] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## New Feature
- [x] The PR creator has listed user actions to use when testing the new feature

### CODE REVIEW

File: packages/app-content-pages/src/helpers/logger/initializeSentry.js [hunk 1]
```diff
@@ -1,18 +1,24 @@
 import * as Sentry from '@sentry/node'
+import { Integrations } from '@sentry/tracing'
 
-export default function initializeSentry () {
+export default function initializeSentry() {
+  const isBrowser = typeof window !== 'undefined'
   const dsn = process.env.SENTRY_CONTENT_DSN
   const release = process.env.COMMIT_ID
   const environment = process.env.APP_ENV
+  const integrations = isBrowser ? [new Integrations.BrowserTracing()] : [new Integrations.Express()]
```
Comment (goplayoutside3) [inline packages/app-content-pages/src/helpers/logger/initializeSentry.js:9]: By enabling Integrations.BrowserTracing() without any of the configuration options, what types of data will we expect to see collected by Sentry? I noticed there's a lot of [config options](https://docs.sentry.io/platforms/javascript/performance/instrumentation/automatic-instrumentation/#configuration-options), and if none of them are used in the NextJS apps, will the amount of tracing data collected be overwhelming?

Comment (goplayoutside3) [reply packages/app-content-pages/src/helpers/logger/initializeSentry.js:9]: For instance, I ran this branch locally and visited the i-fancy-cats project. I see that here: https://zooniverse-27.sentry.io/performance/?project=1492691&statsPeriod=24h

Is that the type of info we're looking to collect?

Comment (eatyourgreens) [reply packages/app-content-pages/src/helpers/logger/initializeSentry.js:9]: I’ve used the same config that we’re using in PFE, so we should collect the same info that we’re already collecting there.

Comment (eatyourgreens) [reply packages/app-content-pages/src/helpers/logger/initializeSentry.js:9]: Core Web Vitals are useful to know, as a measure of how good the volunteer experience is, but right now I think we’re most interested in detecting slow-loading JS files. 

Here’s an example from PFE. 
https://zooniverse-27.sentry.io/issues/3964620701/?project=5623615&query=is%3Aunresolved&referrer=issue-stream&stream_index=23

Comment (eatyourgreens) [reply packages/app-content-pages/src/helpers/logger/initializeSentry.js:9]: Here’s our PFE Sentry config. 
https://github.com/zooniverse/Panoptes-Front-End/blob/521c0f107103bb092a8e441a31910379c504a909/app/lib/init-sentry.js#L9-L16

Comment (goplayoutside3) [reply packages/app-content-pages/src/helpers/logger/initializeSentry.js:9]: Makes sense thanks!

**Review by goplayoutside3 — APPROVED** [review]
I think this looks good. I can run this branch locally, visit i-fancy-cats and then view the Sentry tracing [here](https://zooniverse-27.sentry.io/performance/fem-app-project:0537e0a1f80f4a9c97be050a11e6ebb6/?display=duration&project=1492691&query=release%3Aa8118b130582620db36cd5086e030f4e07ecb68b&referrer=performance-transaction-summary&statsPeriod=14d&transaction=%2Fprojects%2Fbrooke%2Fi-fancy-cats&unselectedSeries=p100%28%29). It includes LCP and Time to First Byte which is helpful performance info. It also includes almost every request that one would see in the Network tab of Browser Tools when loading an FEM page.

 We might want to keep an eye out for expected and unexpected behavior in Sentry logging considering that Sentry.init is now called with tracing multiple times. It's first initiated in _document.js and then again in _app.js.

**Pull request merged**